internal/pkg/cli/env_deploy.go

```diff
@@ -133,8 +149,32 @@ func (o *deployEnvOpts) Ask() error {
 	return o.validateOrAskEnvName()
 }
 
+func validateEnvVersion(vg versionGetter, name, templateVersion string) error {
+	envVersion, err := vg.Version()
```

Comment:
what about if the version is "bootstrap"?

Reply:
Good call we need to deal with this case too.